Extensions/satsukimous.js

```diff
@@ -1,6 +1,6 @@
 //* TITLE Satsukimous **//
-//* VERSION 1.0.4 **//
-//* DESCRIPTION Turns anon icons into Satsuki **//
+//* VERSION 1.1.0 **//
+//* DESCRIPTION Turns anon icons into Satsuki, Ryuko or Mako or a custom image **//
```

Comment:
It's a minor grammar nitpick, but this should be "Turns anon icons into Satsuki, Ryuko, Mako or a custom image"